### DISCUSSION TITLE
Allow adding more than 1 action button to each Notification

### DISCUSSION THREAD

**Discussion by ziming** [laravel/nova-issues#5254] [Q&A] [answered]
Allow adding more than 1 action button to Notifications would be great feature to have, like a 'View' button & a 'Call' button

I would imagine a new chainable method called actions() that accepts an array of NovaNotificationAction objects

**Comment by crynobone** [comment, marked as answer]
Feel free to add your own Notification class and vue component responding to above requirements.
